### DISCUSSION TITLE
It's work with 6700XT but VRAM doesn't clear when is done untill exit and more

### DISCUSSION THREAD

**Discussion by PleaseGIVEitTOme** [lshqqytiger/stable-diffusion-webui-directml#19] [General]
Hello! It's great that i can run it on AMDs GPU (I'm running with 6700XT), but there are 2 problems that i have
1. After the work is done, it doesn't return the VRAM
2. Can't adjust pixel resolution to over 512x512 with a lot prompts due to VRAM running out all the times, but i have a 1070 which is in the same settings. it can do a lot better
However, im waiting for your work update with fixed bug and thanks for hardwork my friend!
*my english not good

**Comment by morpheusseu** [comment]
to Q1, maybe you just shut down the web ui but not close the termination?
to Q2, yes, NVIDIA card is much better for it, which speed up the generation in hardware level (CUDA)

**Reply by PleaseGIVEitTOme** [reply]
Q1 : no man i mean the program won't return my VRAM after generating the image done, but i didn't close the UI because i wanted to continue.
Q2 : About that i know right Nvidia support by Hardware level, but i have no idea why can't go more 512x512 It use my VRAM so high and not return my VRAM for available ( which the 1070 that i have used, it is better )
Well, even though i'm waiting for an update 
Thanks my freind

**Comment by Miraihi** [comment]
DirectML VRAM usage and garbage collection is very bad right now. Don't even compare it with the CUDA options. It's practically the translation layer.
With my RX580 8Gb I practically can't generate anything decent without these arguments:
`COMMANDLINE_ARGS=--medvram --precision full --no-half --no-half-vae --opt-sub-quad-attention --opt-split-attention-v1 --disable-nan-check`
But still the developer's effort looks promising and I'm looking forward for the improvements.

**Reply by PleaseGIVEitTOme** [reply]
Right my friend
I'm waiting for an update, about VRAM usage

**Reply by Mangosniper** [reply]
Anything new here? I just tried again, but it seems the issue is stil present.
